app/src/sharedTest/java/org/oppia/android/app/policies/PoliciesFragmentTest.kt

```diff
@@ -0,0 +1,255 @@
+import org.oppia.android.testing.robolectric.RobolectricModule
+import org.oppia.android.testing.threading.TestDispatcherModule
+import org.oppia.android.testing.time.FakeOppiaClockModule
+import org.oppia.android.util.accessibility.AccessibilityTestModule
+import org.oppia.android.util.caching.AssetModule
+import org.oppia.android.util.caching.testing.CachingTestModule
+import org.oppia.android.util.gcsresource.DefaultResourceBucketName
+import org.oppia.android.util.gcsresource.GcsResourceModule
+import org.oppia.android.util.locale.LocaleProdModule
+import org.oppia.android.util.logging.LoggerModule
+import org.oppia.android.util.logging.firebase.FirebaseLogUploaderModule
+import org.oppia.android.util.networking.NetworkConnectionDebugUtilModule
+import org.oppia.android.util.networking.NetworkConnectionUtilDebugModule
+import org.oppia.android.util.parser.html.HtmlParser
+import org.oppia.android.util.parser.html.HtmlParserEntityTypeModule
+import org.oppia.android.util.parser.image.GlideImageLoaderModule
+import org.oppia.android.util.parser.image.ImageParsingModule
+import org.robolectric.annotation.Config
+import org.robolectric.annotation.LooperMode
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/** Tests for [PoliciesFragment]. */
+@RunWith(AndroidJUnit4::class)
+@LooperMode(LooperMode.Mode.PAUSED)
+@Config(
+  application = PoliciesFragmentTest.TestApplication::class,
+  qualifiers = "port-xxhdpi"
+)
+class PoliciesFragmentTest {
+  @get:Rule
+  val initializeDefaultLocaleRule = InitializeDefaultLocaleRule()
+
+  @Inject
+  lateinit var htmlParserFactory: HtmlParser.Factory
+
+  @Inject
+  @field:DefaultResourceBucketName
+  lateinit var resourceBucketName: String
+
+  @get:Rule
+  var activityTestRule: ActivityTestRule<PoliciesActivity> = ActivityTestRule(
+    PoliciesActivity::class.java, /* initialTouchMode= */
+    true, /* launchActivity= */
+    false
+  )
+
+  @Before
+  fun setUp() {
+    setUpTestApplicationComponent()
+  }
+
+  @Test
+  fun testPrivacyPolicyFragment_checkPrivacyPolicy_isDisplayed() {
+    launch<PoliciesActivity>(createPoliciesActivity(PolicyPage.PRIVACY_POLICY)).use {
+      onView(withId(R.id.policies_description_text_view)).perform(scrollTo())
+        .check(matches(isDisplayed()))
+    }
+  }
+
+  @Test
+  fun testPrivacyPolicyFragment_checkPrivacyPolicyWebLink_isDisplayed() {
+    launch<PoliciesActivity>(createPoliciesActivity(PolicyPage.PRIVACY_POLICY)).use {
+      onView(withId(R.id.policies_web_link_text_view)).perform(scrollTo())
+        .check(matches(isDisplayed()))
+    }
+  }
+
+  @Test
+  fun testPrivacyPolicyFragment_checkTermsOfService_isDisplayed() {
+    launch<PoliciesActivity>(createPoliciesActivity(PolicyPage.TERMS_OF_SERVICE)).use {
+      onView(withId(R.id.policies_description_text_view)).perform(scrollTo())
+        .check(matches(isDisplayed()))
+    }
+  }
+
+  @Test
+  fun testPrivacyPolicyFragment_checkTermsOfServiceWebLink_isDisplayed() {
```

Comment:
Suggest for this & the counterpart test above to verify that the actual link is correct.

Reply:
Done.